kind/common.sh

```diff
@@ -21,17 +21,17 @@ set -ex
 set -o pipefail
 
 _registry_port="5001"
-_registry_name="kind-registry"
+_registry_name="registry"
 
 CTR_CMD=${CTR_CMD-docker}
 PLATFORM=${PLATFORM:-"kind"}
 CONFIG_PATH="kind"
 KIND_VERSION=${KIND_VERSION:-0.17.0}
 KIND_MANIFESTS_DIR="$CONFIG_PATH/manifests"
 CLUSTER_NAME=${KIND_CLUSTER_NAME:-kind}
-REGISTRY_NAME=${REGISTRY_NAME:-kind-registry}
+REGISTRY_NAME=${REGISTRY_NAME:-registry}
 REGISTRY_PORT=${REGISTRY_PORT:-5001}
-KIND_DEFAULT_NETWORK="kind"
+DEFAULT_NETWORK="cluster"
```

Comment:
I suppose default network keep same with kind and I will update in https://github.com/sustainable-computing-io/local-dev-cluster/pull/20

Reply:
k... it used in microshift as well, keep it as cluster.